### ISSUE TITLE
Hawaiian Language Support

### ISSUE THREAD

**Issue by ian4uh** [espeak-ng/espeak-ng#578]
The Hawaiian Language has it's own ISO language code, but it isn't supported, as far as I know, by any screen readers.

- https://en.wikipedia.org/wiki/Hawaiian_language
- https://en.wikipedia.org/wiki/Hawaiian_alphabet

As most Street names in Hawaiʻi are Hawaiian words this can become very confusing. Even the word "Hawaiʻi" when spelled correctly is almost unrecognizable.

Microsoft includes Hawaiian as a Keyboard language in Windows 10, and Apple includes it as an OS language for the entire system. With more people being able to write in the Hawaiian Language, the lack of screen reader support becomes more and more of an issue.

What would be needed to allow proper pronunciation of Hawaiian words?

**Comment by valdisvi** [comment]
Look at [Adding or Improving a Language](https://github.com/espeak-ng/espeak-ng/blob/master/docs/add_language.md).

**Comment by valdisvi** [comment]
@ian4uh, I have added initial support for Hawaiian language. You can test it [online](https://odo.lv/Espeak) and improve it yourself by following [contributor guide](https://github.com/espeak-ng/espeak-ng/blob/master/docs/contributing.md).
